saleor/graphql/checkout/types.py

```diff
@@ -531,7 +531,14 @@ class Checkout(ModelObjectType[models.Checkout]):
         "saleor.graphql.account.types.Address",
         description="The shipping address of the checkout.",
     )
-    note = graphene.String(required=True, description="The note for the checkout.")
+    customer_note = graphene.String(
+        required=True, description="The customer note for the checkout."
+    )
+    note = graphene.String(
```

Comment:
This needs additional resolver, to return a value from `checkout.customer_note` field

Reply:
@korycins  I think it is not True, in the first version we still update both fields. Furthermore, if migration which copy note -> to customer note will be in progress, then we can return the default value instead of the proper one. Can you please check again if this change is needed?

Reply:
>  the first version

What is a "first version" ? I see only PR to main so assume that this is first version.

You have two fields. Old note, and new customerNote. As this is only field renaming, then both of them should return the same value. So during migrating the data from note to customerNote, fetching customerNote could return different values. So I think, both of the resolvers in that case should relay on both fields. At least temporary until migrating the data.  **BUT** This approach may be debatable in that case as we are changing something that wasn't available for modyfing before **3.21** (checkoutCustomerNoteUpdate was added to main only afaik). 

Then we can assume that all existing checkouts don't have any value in `note` field, so in that case `note`'s resolver could immediately return the value for `customerNote`. 

For both approaches, in my opinion, we should have a data migration in celery task - just to keep in mind that there is a possibility of having millions of checkouts so checking the all records could take some time.

Reply:
We do not proceed with migrations for now, so zero downtime migrations will not be utilized.